packages/editor/src/components/projects/ProjectsPage.tsx

```diff
@@ -503,10 +508,10 @@ const ProjectsPage = () => {
                   type="button"
                   variant="contained"
                   color="primary"
-                  disabled={projectState.refreshingGithubRepoAccess.value}
+                  disabled={githubRepoAccessRefreshQuery.data}
                   onClick={() => refreshGithubRepoAccess()}
                 >
-                  {projectState.refreshingGithubRepoAccess.value ? (
+                  {githubRepoAccessRefreshQuery.data ? (
```

Comment:
Here again since .data is an array. We should put this disabled based on .data.length